src/MUSI6106Exec/MUSI6106Exec.cpp

```diff
@@ -127,13 +173,451 @@ int main(int argc, char* argv[])
 
 }
 
-
 void     showClInfo()
 {
     cout << "MUSI6106 Assignment Executable" << endl;
     cout << "(c) 2014-2022 by Alexander Lerch" << endl;
-    cout  << endl;
+    cout << endl;
 
     return;
 }
 
+bool test1();
+void test2();
+bool test3();
+bool test4();
+bool test5();
+
+void runTests()
+{
+    bool test1passed = test1();
+    test2();
+    bool test3passed = test3();
+    bool test4passed = test4();
+    bool test5passed = test5();
```

Comment:
Seems that there's no way to tell if tests are passed or not from the `stdout` or `stderr`? I'm not sure if this is intentional design but it might help if it can print whether each test is passed or not.